x-pack/solutions/security/plugins/security_solution/common/api/endpoint/model/schema/common.schema.yaml

```diff
@@ -244,6 +244,8 @@ components:
 
     ResponseActionDetails:
        type: object
+       required:
+         - command
```

Comment:
Q: There should be several other fields that should be required here. Why is only `command` listed here? In other words what makes command the only _discriminator_ here? Shouldn't `agentType` also be listed within required fields?